sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -381,12 +381,12 @@ querySpecification
     ;
 
 hint
-    : '/*+' hintStatement '*/'
+    : '/*+' hintStatements+=hintStatement (hintStatements+=hintStatement)* '*/'
```

Comment:
In the same block `/*+ */`, multiple hints are separated by commas in Hive. However, in Oracle, it is separated by spaces.